implementations/micrometer-registry-signalfx/src/main/java/io/micrometer/signalfx/SignalFxNamingConvention.java

```diff
@@ -66,9 +74,16 @@ public String tagKey(String key) {
         conventionKey = START_UNDERSCORE_PATTERN.matcher(conventionKey).replaceAll(""); // 2
         conventionKey = SF_PATTERN.matcher(conventionKey).replaceAll(""); // 2
 
+        conventionKey = PATTERN_TAG_KEY_BLACKLISTED_CHARS.matcher(conventionKey).replaceAll("_");
         if (!START_LETTERS_PATTERN.matcher(conventionKey).matches()) { // 3
             conventionKey = "a" + conventionKey;
         }
+        if (PATTERN_TAG_KEY_BLACKLISTED_PREFIX.matcher(conventionKey).matches()) {
+            logger.log("'" + conventionKey + "' (original name: '" + key + "') is not a valid meter name. "
```

Comment:
meter → tag

Reply:
@shakuzen Thanks! I fixed it in 034f52c.